docs/source/getting-started/security-basics.rst

```diff
@@ -146,41 +184,69 @@ itself, ``jupyterhub_config.py``, as a binary string:
    If the cookie secret value changes for the Hub, all single-user notebook
    servers must also be restarted.
 
+.. _cookies:
 
-.. _authentication-token:
+Cookies used by JupyterHub authentication
+-----------------------------------------
 
-Proxy authentication token
---------------------------
+The following cookies are used by the Hub for handling user authentication.
 
-The Hub authenticates its requests to the Proxy using a secret token that
-the Hub and Proxy agree upon. The value of this string should be a random
-string (for example, generated by ``openssl rand -hex 32``).
+jupyterhub-hub-login
+~~~~~~~~~~~~~~~~~~~~
 
-Generating and storing token in the configuration file
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+This is the login token used when visiting Hub-served pages encrypted such
+as the main home, the spawn form, etc. If this cookie is set, then the
+user is logged in.
 
-Or you can set the value in the configuration file, ``jupyterhub_config.py``:
+Resetting the Hub cookie secret effectively revokes this cookie.
 
-.. code-block:: python
+This cookie is restricted to the path ``/hub/``.
 
-    c.JupyterHub.proxy_auth_token = '0bc02bede919e99a26de1e2a7a5aadfaf6228de836ec39a05a6c6942831d8fe5'
+jupyterhub-user-<username>
+~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Generating and storing as an environment variable
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+This is the cookie used for authenticating with a single-user server
+encrypted. It is set by the single-user server after OAuth with the Hub.
```

Comment:
another stray "encrypted"

Reply:
Oh, in this case I think "encrypted" can be removed. Updating it now, thanks!